test/TestSetup/InMemoryAssembler.cpp

```diff
@@ -129,12 +130,16 @@ InMemoryObject::InMemoryObject(const char* source, const char* cpu, const char**
     for(auto sit = object->sections().begin(); sit != object->sections().end(); ++sit)
     {
         if(sit->isText()) {
-            if((ec = sit->getContents(text_section))) {
-                llvm::errs() << "Failed to load text section: " << ec.message() << "\n";
-                return;
-            }
-            else {
+            if (llvm::Expected<llvm::StringRef> E  = sit->getContents()) {
+                text_section = *E;
                 break;
+            } else {
+                llvm::handleAllErrors(E.takeError(), [&](const llvm::ErrorInfoBase &EIB) {
```

Comment:
Why do you need to capture with ``[&]`` ?

Reply:
No need or the ``[&]``. I copy some llvm code that handle error and forget to remove the capture.